packages/lit-dev-content/site/docs/libraries/labs.md

```diff
@@ -6,14 +6,26 @@ eleventyNavigation:
   order: 3
 ---
 
-Lit Labs is an umbrella for projects related to Lit that aren't quite ready for production. These projects may be experimental or incomplete, and they're subject to breaking changes. Lit Labs projects are published under the `@lit-labs` npm scope.
+Lit Labs is an umbrella for projects related to Lit that may be experimental or incomplete. Breaking changes are likely to occur more frequently than in non-labs packages, but they will still respect standard semantic versioning guildelines and all changes will be published to the CHANGELOG files. Lit Labs projects are published under the `@lit-labs` npm scope.
```

Comment:
```suggestion
Lit Labs is an umbrella for projects related to Lit that may be experimental or incomplete. Breaking changes are likely to occur more frequently than in non-labs packages, but they will still respect standard semantic versioning guidelines and all changes will be published to the CHANGELOG files. Lit Labs projects are published under the `@lit-labs` npm scope.
```

Reply:
I think it might read better (and not sound so scary) if we started out with a more positive tone, and then listed the caveats as a bulleted list so it's not such a wall of text, ala something like:

> Lit Labs is an umbrella for Lit packages under development that we are actively seeking feedback on. While we encourage real-world use in order to help the feedback process, please note the following important aspects of Labs packages:
>
> * Breaking changes are likely to occur more frequently...<snip>
>* When a Lit Labs project is ready to be graduated, it will begin publishing under the `@lit` scope...<snip>
>* We may decide to deprecate a Lit Labs project...<snip>
>
> Feedback is currently being solicited on the following Labs packges:

Reply:
Good call on making it a bulleted list.